homematic/data/run.sh

```diff
@@ -37,11 +35,11 @@ init_interface_list "$RF_ENABLE" "$HMIP_ENABLE" "$WIRED_ENABLE"
 # RF support
 if [ "$RF_ENABLE" == "true" ]; then
     for (( i=0; i < "$RF_DEVICES"; i++ )); do
```

Comment:
This for loop can be changed to a loop over the keys. 

See example here: <https://github.com/hassio-addons/addon-ftp/blob/master/ftp/rootfs/etc/cont-init.d/users.sh#L9>

This eliminates the use of a length based variable as well.